index.html

```diff
@@ -234,30 +243,34 @@ <h1 id="header-text">
-                  <li><a href="" class="frm-btn"><img src="./images/Frame html.svg" alt="html-frame">  </a></li>
-                  <li><a href="" class="frm-btn"><img src="./images/Frame css.svg" alt="css-frame">  </a>   </li>
+            <div class="languages-frameworks">
+              <ul class="lang-frmwrk-ul" id="js-html-css-frames">
+                  <li id="js-frm"><a href="" class="frm-btn"><img src="./images/Frame JS-frame.svg" alt="JavaScript-frame" id="js-frm-img"> </a></li>
+                  <li id="desk-js-frm" class="desk-js-frm"><a href="" class="desk-frm-btn"><img src="./images-desktop/js-Frame.png" alt="JavaScript-frame" id="desk-js-frm-img"> </a></li>
+                  <li id="html-frm"><a href="" class="frm-btn"><img src="./images/Frame html.svg" alt="html-frame">  </a></li>
+                  <li id="desk-html-frm" class="desk-js-frm" ><a href="" class="desk-frm-btn"><img src="./images-desktop/html-Frame.png" alt="html-frame" id="desk-js-frm-img-1">  </a></li>
+                  <li id="css-frm" ><a href="" class="frm-btn"><img src="./images/Frame css.svg" alt="css-frame">  </a>   </li>
+                  <li id="desk-css-frm" class="desk-js-frm"><a href="" class="desk-frm-btn"><img src="./images-desktop/Css-Frame.png" alt="css-frame" id="desk-js-frm-img-2">  </a>   </li>
                   <li></li>
               </ul>
             </div> 
             <div class="frameworks">
               <ul class="lang-frmwrk-ul">
-                <li><a href="" class="frameworks"> <b>Frameworks </b><b>&gt;</b> </a> <br>
+                <li><a href="" class="frameworks" id="frame"> <b>Frameworks </b><b>&gt;</b> </a> <br>
                   <img src="./images/Seperator-line.svg" alt="seperator"></li>
-                <li><a href="" class="frameworks skills"> <b>Skills</b><b>&gt;</b></a>
-                <img src="./images/Seperator-line.svg" alt="seperator"></li>
+                <li><a href="" class="frameworks skills" id="skills"> <b>Skills</b><b>&gt;</b></a>
+                <img src="./images/Seperator-line.svg" alt="seperator" id="skills-img"></li>
               </ul>
             </div>
         </div>
         <div id="contact-me">
           <img src="./images/Ellipse 3.png" alt="eclipse-3" id="ellipse-3">
+          <img src="./images-desktop/Desk-Ellipse.png" class="desk-ellipse" alt="eclipse-3" id="desk-ellipse-5">
             <h2 id="contact-me-h2">Contact me</h2>
-            <p id="contact-me-text">If you have and application you <br> 
-                are interested in developing, a <br> 
-                feature that you need built or a <br>
-                project that needs coding. <br>
+            <p id="contact-me-text">If you have and application you 
+                are interested in developing, a 
+                feature that you need built or a
+                project that needs coding.
                 I'd love to help with it.
             </p>
```

Comment:
- [ ] Please make sure that your contact form design on the desktop matches the Figma design :pray: As you can see below, In your design, the width of the input fields is not the same as it is in the Figma design :blush: 
  
  ### Your design
    ![](https://www7.lunapic.com/do-not-link-here-use-hosting-instead/168081658857020455?71957867880)
  
  ### Figma design
    ![image](https://user-images.githubusercontent.com/92303891/230499790-1d3b014d-bc75-44d7-afce-77013946af5c.png)